packages/components/src/globals/scss/_typography.scss

```diff
@@ -48,3 +48,49 @@ $base-font-size: 16px !default;
 @mixin type-style($name, $fluid: false) {
   @include carbon--type-style($name, $fluid);
 }
+
+//default type styles
+h1 {
+  @include reset;
+  @include type-style('productive-heading-05');
+}
+
+h2 {
+  @include reset;
+  @include type-style('productive-heading-04');
+}
+
+h3 {
+  @include reset;
+  @include type-style('productive-heading-03');
+}
+
+h4 {
+  @include reset;
+  @include type-style('productive-heading-02');
+}
+
+h5 {
+  @include reset;
+  @include type-style('productive-heading-01');
+}
+
+p {
+  @include reset;
+  @include type-style('body-long-02');
+}
+
+a {
+  @include reset;
+  color: $link-01;
+}
+
+em {
+  @include reset;
+  font-style: italic;
+}
+
+strong {
+  @include reset;
+  font-weight: 700;
```

Comment:
We have a type token to use here instead of `700`. Also, we shouldn't use `700`. See https://www.carbondesignsystem.com/guidelines/typography/overview#weights.

Reply:
What type token would pertain to this? There didn't seem to be any tokens for it before so I didn't think there was one now, but can definitely update if there is. I was just setting the default styles to reflect the migrated versions of v9.

Reply:
`@include carbon--font-weight('semibold');`

Docs: https://github.com/carbon-design-system/carbon/blob/master/packages/type/docs/sass.md#carbon--font-weight-mixin